app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductFragment.java

```diff
@@ -265,6 +267,12 @@ private void setupViewPager(ViewPager viewPager) {
         }
         if (BuildConfig.FLAVOR.equals("off")) {
             adapterResult.addFragment(new NutritionProductFragment(), menuTitles[2]);
+            if( mState.getProduct().getNutriments() != null && mState.getProduct().getNutriments().contains(Nutriments.CARBON_FOOTPRINT) )
```

Comment:
This also can be done in a simple `if` statement like the others.

Reply:
Yes, I'll fix this the same way.